EventListener/SecurityListener.php

```diff
@@ -80,7 +83,7 @@ public function onKernelControllerArguments(FilterControllerArgumentsEvent $even
     // code should be sync with Symfony\Component\Security\Core\Authorization\Voter\ExpressionVoter
     private function getVariables(FilterControllerArgumentsEvent $event)
     {
-        $request = $event->getArguments();
+        $request = $event->getRequest();
```

Comment:
This was a bug in my original code, unrelated to this PR. A test now covers this.